contract_manager/scripts/upgrade_evm_entropy_contracts.ts

```diff
@@ -0,0 +1,74 @@
+import yargs from "yargs";
```

Comment:
This is 99% similar to the other upgrade script. Please refactor as much as you can